### PR TITLE
Cctri 1917 - Code restructure and configuration token support 

### PR DESCRIPTION
**Pull request by TarasMaliuchenko**

### CODE REVIEW

File: module_type.json.sample [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      "key": "custom_CTR_ENTITIES_LIMIT",
+      "type": "integer",
+      "label": "Entities Limit",
+      "tooltip": "Restricts the maximum number of `Sightings`",
+      "required": false
+    },
+    {
+      "key": "custom_AGGREGATE",
+      "type": "boolean",
+      "label": "Entities Limit",
```
Comment (AnastasiiaRozlyvan) [inline module_type.json.sample:35]: I think there is something wrong with this label

Comment (TarasMaliuchenko) [reply module_type.json.sample]: Fixed

File: module_type.json.sample [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  "tips": "When configuring this integration, you must first gather some information from your Farsight Security DNSDB® account, then add the Farsight Security DNSDB® Module\n\n1. Locate your **DNSDB API key**, it may be found in an email from Farsight Security\n2. Complete the **Add New Farsight Security DNSDB® Module** form:\n    - **Module Name** - Leave the default name or enter a name that is meaningful to you.\n    - **API Key** - Enter the API Key from your Farsight Security DNSDB® account information.\n3. Click **Save** to complete the Farsight Security DNSDB® module configuration",
```
Comment (AlinaBlahun) [inline module_type.json.sample:6]: Should there be something about CTR_ENTITIES_LIMIT and AGGREGATE?

Comment (TarasMaliuchenko) [reply module_type.json.sample]: Fixed

**Review by AlinaBlahun — APPROVED** [review]

**Review by AnastasiiaRozlyvan — APPROVED** [review]

**Pull request merged**